### PR TITLE
Create a fetch module to hold all data fetch functions

### PR DESCRIPTION
**Pull request by shifucun**
There are quite some duplicate functions to fetch data (and with similar response data structure).

This is to consolidate and have all data fetch functions into one "fetch" module. This makes it easier to test, inspect and do data conversion for API update.

### CODE REVIEW

File: server/lib/fetch.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  """Returns full property values data out of REST API response."""
+  resp = dc.property_values(nodes, prop, out)
+  result = {}
+  for node, node_arcs in resp.get('data', {}).items():
+    result[node] = node_arcs.get('arcs', {}).get(prop, {}).get('nodes', [])
+  return result
+
+
+def triples(nodes, out=True):
+  """Fetch triples for given nodes"""
```
Comment (chejennifer) [inline server/lib/fetch.py:190]: nit: add function comments for what the return object looks like for all the functions in this module

**Review by chejennifer — APPROVED** [review]
nice, this is very clean!

File: server/services/datacommons.py [hunk 2]
```diff
@@ -221,7 +221,7 @@ def triples(nodes, direction):
   return post(url, {
       'nodes': nodes,
       'property': '->*' if direction == 'out' else '<-*'
-  }).get('data', {})
+  })
```
[the remark this answers is not in the document]
Comment (shifucun) [reply server/services/datacommons.py:224]: Done. And add a fetch.properties() function

**Review by shifucun — COMMENTED** [review]
Thanks for the review!

**Pull request merged**